requirements.txt

```diff
@@ -1 +1,5 @@
-requests
+requests==2.24.0
+argparse==1.4.0
```

Comment:
Argparse is part of the Python Standard Library and we don't need a specific version.

Running `pipreqs` on my dev system yielded the following requirements file:
```
urllib3==1.25.8
requests==2.22.0
```

Reply:
I have removed the argparse :+1: